### ISSUE TITLE
Game not launching anymore with Smart

### ISSUE THREAD

**Issue by CoolHandDevil505** [Algalish/SmartIssueTracker#121]
Hi!

Do not know since wich game/mod version, but I cannot launch the game (experimental or early access) with the mod installed. I tried several versions of the mod with no luck. I do not have a crash report nor an error message…
It synchronises cloud, never show the Satisfactory logo, and syncs again…
Using SMM 2.3.1.

Thank you for this smartvelous mod.

**Comment by thadius856** [comment]
Same issue for me. The process seems to start briefly, but writes nothing to the log files, then immediately quit before showing anything on screen.

Experimental CL143604
SML v2.2.1
Smart! 0.16.0 (2021-02-16 18:53:30 PM)

**Comment by Algalish** [comment]
@thadius856 
>Experimental CL143604
You are using experimental branch, the mods are not working on it yet.

@CoolHandDevil505 
Are you sure the you are playing Early Acess and not Experimental?

**Comment by amoriarty** [comment]
Hi,
I also have this issues since version 0.15.0. I find a quick way to "ficsit" today by going to SMM settings and in debug menu use the clear cache function.

Hopes that can help.

**Comment by Algalish** [comment]
@amoriarty  You are on Early Access, not Experimental, right?

**Comment by amoriarty** [comment]
@Algalish Yes, I'm on Early Access. Didn't try with Experimental.

**Comment by CoolHandDevil505** [comment]
> @CoolHandDevil505
> Are you sure the you are playing Early Acess and not Experimental?

I have not playing on EA since update III, and your mod worked fine until maybe version 12 or 13. Then I stop playing for a few weeks (a shame I know), and discover what described earlier. I tried immediately launching on EA, but apparently had an issue with my cloud saves, and did not see it because I launched from SMM. And I did not know that the mod was not for Exp branch. So I am sorry for this false positive and for the time you wasted on this. :-/
